database/src/main/scala/cromwell/database/slick/SlickDatabase.scala

```diff
@@ -667,4 +668,61 @@ class SlickDatabase(databaseConfig: Config) extends SqlDatabase {
 
     runTransaction(action)
   }
+
+  override def addMetadataEvent(workflowUuid: String,
+                                key: String,
+                                value: String,
+                                timestamp: Timestamp)(implicit ec: ExecutionContext): Future[Unit] = {
+
+    val action = DBIO.seq(dataAccess.metadataAutoInc += Metadatum(workflowUuid, key, callFqn = None, index = None, attempt = None, Option(value), timestamp))
```

Comment:
I think this might be the first time I've ever seen an actual footnote in a code review comment. Bravo @kshakir - you win the internets today.